modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,204 @@
+//  (link:https://github.com/tektoncd/triggers/pull/1113[#1113])
++
+[IMPORTANT]
+====
+Breaking change: The deprecated `replicas` field has been removed.
+====
+
+[id="deprecated-features-pipelines-operator-1-6_{context}"]
+=== {pipelines-title} Operator
+
+* In {pipelines-title} 1.6, the values of `HOME="/tekton/home"` and `workingDir="/workspace"` are removed from the `Steps` specification.
++
+Instead, {pipelines-title} sets `HOME` and `workingDir` to the values defined by your `Steps` containers. You can override these values in your `Steps` specification.
++
+To use the older behavior, you can change the `disable-working-directory-overwrite` and `disable-home-env-overwrite` fields in the `TektonConfig` CR to `false`:
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+  kind: TektonConfig
+  metadata:
+    name: config
+  spec:
+    pipeline:
+      disable-working-directory-overwrite: false
+      disable-home-env-overwrite: false
+  ...
+----
++
+[IMPORTANT]
+====
+The `disable-working-directory-overwrite` and `disable-home-env-overwrite` fields in the `TektonConfig` CR are now deprecated and will be removed in a future release.
+====
+// (link:https://issues.redhat.com/browse/SRVKP-1465[SRVKP-1465])
+
+[id="known-issues-1-6_{context}"]
+== Known issues
+
+* When you run Maven and Jib-Maven cluster tasks, the default container image is supported only on Intel (x86) architecture. Therefore, tasks will fail on IBM Power Systems (ppc64le), IBM Z, and LinuxONE (s390x) clusters. As a workaround, you can specify a custom image by setting the `MAVEN_IMAGE` parameter value to `maven:3.6.3-adoptopenjdk-11`.
+// issue # is unknown.
+
+* On IBM Power Systems, IBM Z, and LinuxONE, the `s2i-dotnet` cluster task is unsupported.
+// issue # is unknown.
+
+* Before you install tasks based on Tekton Catalog on IBM Power Systems (ppc64le), IBM Z, and LinuxONE(s390x) using `tkn hub`, verify if the task can be executed on these platforms. To check if `ppc64le` and `s390x` are listed in the "Platforms" section of the task information, you can run the following command: `tkn hub info task <name>`
+// issue # is unknown.
+
+* You cannot use the `nodejs:14-ubi8-minimal` image stream because doing so generates the following errors:
++
+[source,terminal]
+----
+STEP 7: RUN /usr/libexec/s2i/assemble
+/bin/sh: /usr/libexec/s2i/assemble: No such file or directory
+subprocess exited with status 127
+subprocess exited with status 127
+error building at STEP "RUN /usr/libexec/s2i/assemble": exit status 127
+time="2021-11-04T13:05:26Z" level=error msg="exit status 127"
+----
+// https://issues.redhat.com/browse/SRVKP-1782
+
+
+Workaround: <workaround>.
+
+[id="fixed-issues-1-6_{context}"]
+== Fixed issues
+
+* The `tkn hub` command is now supported on IBM Power Systems, IBM Z, and LinuxONE.
+// issue # is unknown.
+
+[id="fixed-cli-0-21-0-1-6_{context}"]
+=== CLI
+
```

Comment:
There is one behavior change that we want to point out for the CLI (`tkn`)

Previously, when any user would delete all resources using `tkn` (example: `tkn pipelinerun delete --all`), then all the resources regardless of their running status would get deleted, but in 1.6.0, the resources that are still in `RUNNING` state do not get deleted. This is reflected in the user facing output as well when the `tkn` command is run. This was tracked in https://issues.redhat.com/browse/SRVKP-1638

Reply:
In addition to the RN, the note should also be added here: https://docs.openshift.com/container-platform/4.9/cli_reference/tkn_cli/op-tkn-reference.html#pipelinerun-delete. I will take care up updating the link above.

Reply:
Please review the new item for this on line 201 of modules/op-release-notes-1-6.adoc